client/src/Components/Home/Members/index.jsx

```diff
@@ -1,5 +1,68 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Row, Col } from 'react-bootstrap';
 
-export default function Members() {
-  return <h1> members cards</h1>;
+import './style.css';
+
+// eslint-disable-next-line react/prop-types
+export default function Members({ filtered, history }) {
+  return (
+    <Row key={Math.random()}>
+      {filtered[0] ? (
+        filtered.map(member => {
+          return (
+            <Col xs={12} md={4} lg={4} key={Math.random()}>
```

Comment:
@Fatmasiam I think you can remove the `key={Math.random()}` here, since there is only one Col inside the parent Row.  The random key will hurt React's performance a little bit, because it will make React think that the whole component needs to be recreated every time render is called (because of the new random key) instead of letting React only replace the inner parts of the component that actually changed.

Reply:
@Fatmasiam can't you use the member_id here?